policy/action.go

```diff
@@ -44,6 +44,9 @@ const (
 	// DeleteBucketPolicyAction - DeleteBucketPolicy Rest API action.
 	DeleteBucketPolicyAction = "s3:DeleteBucketPolicy"
 
+	// DeleteBucketCorsAction - DeleteBucketCors Rest API action.
+	DeleteBucketCorsAction = "s3:DeleteBucketCors"
```

Comment:
Ah one thing missing you need to add these actions to "AllActions" as well as actions with Conditions list

Reply:
Will do